Open Judge System/Web/OJS.Web/Web.config

```diff
@@ -58,6 +58,8 @@
     <add key="ArchiveSingleBatchLimit" value="25000" />
     <add key="ArchiveMaxSubBatchSize" value="10000" />
     <add key="ArchiveDailyBatchSize" value="500000" />
+    <add key="JudgeBaseUrl" value="http://localhost:1721" />
```

Comment:
Why this port, don't we have another for when starting Judge?

Reply:
This is the HTTP port. The different one is for https://localhost:44307
I can leave the HTTPS URL if most team members use it.
I am using it on HTTP and that's why I did it like that.